tools/dep_updaters/README.md

```diff
@@ -35,3 +35,28 @@ been created with the changes), do the following:
    example).
 
 [`c61870c`]: https://github.com/nodejs/node/commit/c61870c376e2f5b0dbaa939972c46745e21cdbdd
+
+## simdutf
```

Comment:
I think there should be a README in the deps/simdutf directory as well that points to this document and/or the script.

Reply:
Added a readme. Thanks for the review.